bytecode/flags.py

```diff
@@ -0,0 +1,297 @@
+# alias to keep the 'bytecode' variable free
+from enum import IntEnum
+from collections import defaultdict
+
+import bytecode as _bytecode
+
+
+class CoFlags(IntEnum):
+    """Possible values of the co_flags attribute of Code object.
+
+    Notes
+    -----
+    We do not rely on inspect values here as some of them are missing and
+    furthermore would be version dependent.
+
+    """
+    CO_OPTIMIZED             = 0x00001  # noqa
+    CO_NEWLOCALS             = 0x00002  # noqa
+    CO_VARARGS               = 0x00004  # noqa
```

Comment:
I dislike hardcoded values. I would prefer to get them from a Python module, like dis.COMPILER_FLAG_NAMES, if possible.